app/assets/javascripts/discourse/app/components/route-control.js

```diff
@@ -0,0 +1,34 @@
+import Component from "@ember/component";
+import { inject as service } from "@ember/service";
+import discourseComputed from "discourse-common/utils/decorators";
+import { defaultHomepage } from "discourse/lib/utilities";
+
+export default Component.extend({
+  tagName: "div",
```

Comment:
I agree, though I thought all components auto-generate divs and wasn't aware of a way to get rid of it... I suppose it may be good to make it a glimmer component to prevent this, but I'm not too sure where we are in using those with Discourse yet..

Reply:
```suggestion
  tagName: "",
```

That will omit the tag in the classic component version. Yes, we can use Glimmer components, there are already a few included now in core, search for `GlimmerComponent`.